### ISSUE TITLE
undefined reference to `major'

### ISSUE THREAD

**Issue by punk-joker** [gittup/tup#441]
I try to build v0.7.3 (need to some tests) and get error:

```
  bootstrap CC -g ../src/compat/utimensat_linux.c
  bootstrap CC -g ../src/sqlite3/sqlite3.c
  bootstrap LD tup -lm
/usr/bin/ld: fuse_server.o: in function `server_init':
/toolchain/tup/build/../src/tup/server/fuse_server.c:208: undefined reference to `major'
/usr/bin/ld: /toolchain/tup/build/../src/tup/server/fuse_server.c:208: undefined reference to `major'
/usr/bin/ld: /toolchain/tup/build/../src/tup/server/fuse_server.c:209: undefined reference to `minor'
/usr/bin/ld: /toolchain/tup/build/../src/tup/server/fuse_server.c:209: undefined reference to `minor'
collect2: error: ld returned 1 exit status
root@d9e41deda525:/toolchain/tup# 
```

Ubuntu 20.04

**Comment by punk-joker** [comment]
I compiled it.  just used hack and removed checking this value.

**Comment by gittup** [comment]
That came up as a result of some glibc changes IIRC, and was fixed in commit 4131ddf5c9f76e1883cd95b006a3818e490a8be8 which wasn't in v0.7.3 (it was first released in v0.7.6). So you could backport that patch as a workaround.
